XamlControlsGallery/ControlPages/GridViewPage.xaml

```diff
@@ -114,9 +288,12 @@
                             Tag="TextTemplate" />
                         <ToggleButton Margin="0,8,0,0" Click="FlowDirectionCheckBox_Click" Content="Reverse FlowDirection" />
```

Comment:
I think it would be nice to also reflect the FlowDirection in the XAML sample. 😅

Reply:
Nice catch! Fixed in upcoming commit.